worlds/sc2/mission_tables.py

```diff
@@ -486,5 +486,5 @@ def get_campaign_potential_goal_missions(campaign: SC2Campaign) -> List[SC2Missi
     return missions
 
 
-def get_no_build_missions() -> List[SC2Mission]:
-    return [mission for mission in SC2Mission if MissionFlag.NoBuild in mission.flags]
+def get_missions_with_flags(flags: MissionFlag) -> List[SC2Mission]:
+    return [mission for mission in SC2Mission if flags & mission.flags]
```

Comment:
Note that there's two syntaxes going on here that only behave the same with one flag set.
* `flagset in mission.flags` returns true if _all_ flags in `flagset` are in `mission.flags`
* `flagset & mission.flags` returns a truthy value if _any_ flag in `flagset` is in `mission.flags`

I'd recommend rewording the function to clarify which behaviour this is, or at least putting it in a docstring.

Reply:
It's _any_ flag in this case, because the function is used to return missions for the exclusions list. I'll reword the name.